include/vast/ABI/Classify.hpp

```diff
@@ -566,10 +657,11 @@ namespace vast::abi
                 if ( hi_start == 8 )
                     return lo;
                 // TODO(abi): float, half -> promote to double
-                VAST_CHECK( TypeConfig::is_scalar_integer( lo ) ||
-                            TypeConfig::is_pointer( lo ),
-                            "Cannot adjust half type for pair {0}. {1}", lo, hi );
-                return TypeConfig::iN( lo, 64 );
+                if (TypeConfig::is_scalar_integer( lo ) || TypeConfig::is_pointer( lo ))
```

Comment:
```suggestion
                if (TypeConfig::is_scalar_integer(lo) || TypeConfig::is_pointer(lo))
```

Reply:
Literally right next condition uses different style

Reply:
Will include a reformat commit at the end of the PR.